src/server/scripts/Northrend/Ulduar/HallsOfLightning/boss_ionar.cpp

```diff
@@ -87,10 +87,10 @@ class boss_ionar : public CreatureScript
             bIsSplitPhase = true;
             bHasDispersed = false;
 
-            uiSplitTimer = 25 * IN_MILLISECONDS;
+            uiSplitTimer = 25s;
 
-            uiStaticOverloadTimer = urand(5 * IN_MILLISECONDS, 6 * IN_MILLISECONDS);
-            uiBallLightningTimer = urand(10 * IN_MILLISECONDS, 11 * IN_MILLISECONDS);
+            uiStaticOverloadTimer = (5s, 6s);
```

Comment:
Instead of using this, you would basically update events or tasks in UpdateAI. And then, schedule these events.

Reply:
This doesn't work, you are always setting it to 6 seconds